pkg/psmdb/statefulset.go

```diff
@@ -446,6 +452,45 @@ func backupAgentContainer(cr *api.PerconaServerMongoDB, replsetName string, tlsE
 	return c
 }
 
+func buildMongoDBURI(ctx context.Context, tlsEnabled bool, sslSecret *corev1.Secret) string {
+	uri := "mongodb://$(PBM_AGENT_MONGODB_USERNAME):$(PBM_AGENT_MONGODB_PASSWORD)@$(POD_NAME)/?replicaSet=$(PBM_MONGODB_REPLSET)"
```

Comment:
we shouldn't use `?replicaSet` in URI, see https://perconadev.atlassian.net/browse/K8SPSMDB-1223